packages/kuma-gui/package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@gera2ld/tarjs": "^0.3.1",
     "@kong-ui-public/app-layout": "^4.3.10",
+    "@kong-ui-public/entities-shared": "^3.17.1",
```

Comment:
It looks like you've installed this but not committed the `package-lock.json` changes?